src/test/resources/features/DeleteTournament.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Tournament
+  In order to use the app
+  As a tournament master
+  I want to delete a tournament
+
+  Scenario: Tournament Master deletes tournament
+    Given I login as "demoTM" with password "password"
+    And  I create a new tournament with name "FirstTournament", level "AMATEUR" and game "LoL"
+    When I delete a tournament with name "FirstTournament", level "AMATEUR" and game "LoL"
+    Then The response code is 204
+    And It does not exist a tournament with name "FirstTournament"
+
+  Scenario: Player tries to delete a tournament
+    Given I login as "demoTM" with password "password"
+    And  I create a new tournament with name "FirstTournament", level "AMATEUR" and game "LoL"
+    Given I login as "demoP" with password "password"
```

Comment:
Although this step has been declared as 'Given', you can use it with 'And', 'Then' or whatever prefix you like for a better wording.